cuda_bindings/docs/source/environment_variables.md

```diff
@@ -0,0 +1,13 @@
+# Environment Variables
+
+## Build-Time Environment Variables
+
+- `CUDA_HOME` or `CUDA_PATH`: Specifies the location of the CUDA Toolkit.
+
+- `CUDA_PYTHON_PARSER_CACHING` : bool, toggles the caching of parsed header files during the cuda-bindings build process. If caching is enabled (`CUDA_PYTHON_PARSER_CACHING` is True), the cache path is set to ./cache_<library_name>, where <library_name> is derived from the cuda toolkit libraries used to build cuda-bindings.
+
+- `CUDA_PYTHON_PARALLEL_LEVEL` (previously `PARALLEL_LEVEL`) : int, sets the number of threads used in the compilation of cython files. This is passed as the `nthreads` argument to :meth:`cython.cythonize` and as the parallel attribute for building extension modules.
```

Comment:
Let's not mention the implementation details and focus on the user-visible behaviors.
```suggestion
- `CUDA_PYTHON_PARALLEL_LEVEL` (previously `PARALLEL_LEVEL`) : int, sets the number of threads used in the compilation of extension modules. Not setting it or setting it to 0 would disable parallel builds.
```